src/common/device_helpers.cuh

```diff
@@ -406,10 +406,14 @@ struct XGBDefaultDeviceAllocatorImpl : XGBBaseDeviceAllocator<T> {
   }
 #if defined(XGBOOST_USE_RMM) && XGBOOST_USE_RMM == 1
   XGBDefaultDeviceAllocatorImpl()
-    : SuperT(rmm::mr::get_current_device_resource(), cudaStream_t{0}) {}
+    : SuperT(rmm::mr::get_current_device_resource(), cudaStream_t{nullptr}) {}
```

Comment:
Is CUDA stream a pointer?

Reply:
Yup.  It's an opaque pointer.

Reply:
Thanks. All this time I thought it was an integer like a file descriptor. I learned something new today.